modules/migration-about-configuring-proxies.adoc

```diff
@@ -0,0 +1,124 @@
+
+For {product-title} 4.2 to {product-version}, the {mtc-full} ({mtc-short}) inherits the cluster-wide proxy settings. You can change the proxy parameters if you want to override the cluster-wide proxy settings.
+
+== Direct volume migration
+
+Direct Volume Migration (DVM) was introduced in MTC 1.4.2. DVM supports only one proxy. The source cluster cannot access the route of the target cluster if the target cluster is also behind a proxy.
+
+If you want to perform a DVM from a source cluster behind a proxy, you must configure a TCP proxy that works at the transport layer and forwards the SSL connections transparently without decrypting and re-encrypting them with their own SSL certificates. A Stunnel proxy is an example of such a proxy.
+
+=== TCP proxy setup for DVM
+
+You can set up direct connection between the source and the target cluster through a TCP proxy and configure the `stunnel_tcp_proxy` variable in the `MigrationController` CR use the proxy:
+
+[source, yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigrationController
+metadata:
+  name: migration-controller
+  namespace: openshift-migration
+spec:
+  [...]
+  stunnel_tcp_proxy: http://username:password@ip:port
+----
+
+Direct volume migration (DVM) only supports Basic Authentication for the proxy. Moreover, DVM only works behind proxies which can tunnel a TCP connection transparently. HTTP/HTTPS proxies in man-in-the-middle mode will not work. The existing cluster-wide proxies may or may not support this behavior. As a result, the proxy settings for DVM are intentionally kept different from usual proxy configuration in {mtc-short}.
+
+=== Why use a TCP proxy instead of an HTTP/HTTPS proxy?
+
+DVM is enabled by running Rsync between the source and the target cluster over an OpenShift Route.  Traffic is encrypted using Stunnel, which in itself is a TCP proxy. The Stunnel running on the source cluster initiates a TLS connection with the target Stunnel and transfers data over an encrypted channel.
+
+Cluster-wide HTTP/HTTPS proxies in OpenShift are usually configured in man-in-the-middle mode where they negotiate their own TLS session with the outside servers. However, this does not work with Stunnel. Stunnel requires that its TLS session be untouched by the proxy, essentially making the proxy a transparent tunnel which simply forwards the TCP connection as-is. Therefore, a TCP proxy must be used.
+
+=== Known Issue
+
+.Migration fails with error `Upgrade request required`
+
+The migration Controller uses the SPDY protocol to execute commands within remote Pods. If the remote cluster is behind a proxy or a firewall that does not support the SPDY protocol, the migration controller  fails to execute remote commands. The migration fails with the error message `Upgrade request required`.
```

Comment:
The migration Controller uses the SPDY protocol to execute commands within remote Pods.

- Please expand **SPDY** in the first usage in the module.
- Pods >> pods (https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting)

Reply:
the migration controller  fails to execute remote commands >>please check spacing between **controller** and **fails.**

Reply:
SPDY is the protocol's name. It is not an acronym.  "The name "SPDY" is a [trademark](https://en.wikipedia.org/wiki/Trademark)[[6]](https://en.wikipedia.org/wiki/SPDY#cite_note-6) of Google and is not an [acronym](https://en.wikipedia.org/wiki/Acronym)" from https://en.wikipedia.org/wiki/SPDY.

Reply:
Other changes made.